packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1342,6 +1371,10 @@ export default class SalesforceCRMService implements CRM {
       this.log.warn(`No uid for booking with organizer ${organizerEmail}`);
     }
 
+    if (fieldValue === DateFieldTypeData.BOOKING_CANCEL_DATE) {
+      return new Date().toISOString();
+    }
```

Comment:
If cancelling a booking and the user wants to write the date that the booking is cancelled, assume that the time that this method is called is the time that the booking cancel request was made.